### PR TITLE
{chem}[foss/2020a, intel/2020a] kim-api v2.1.3

### PR DESCRIPTION
**Pull request by ocaisa**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/9c7bf22767a2e306c84700d9abbb17b8 for a full test report.

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/4219a5c182126be55baf5e4e07da30ac for a full test report.

File: easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-foss-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('CMake', '3.16.4'),  # Also needed to install models, thus not just a builddependency.
+]
+
+source_urls = ['https://s3.openkim.org/kim-api/']
+sources = ['%(name)s-%(version)s.txz']
+checksums = ['88a5416006c65a2940d82fad49de0885aead05bfa8b59f87d287db5516b9c467']
```
Comment (akesandgren) [inline easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-foss-2020a.eb]: Move this block above the dependencies

File: easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-intel-2020a.eb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+dependencies = [
+    ('CMake', '3.16.4'),  # Also needed to install models, thus not just a builddependency.
+]
+
+source_urls = ['https://s3.openkim.org/kim-api/']
+sources = ['%(name)s-%(version)s.txz']
+checksums = ['88a5416006c65a2940d82fad49de0885aead05bfa8b59f87d287db5516b9c467']
```
Comment (akesandgren) [inline easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-intel-2020a.eb]: Move this block above the dependencies

File: easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-intel-2020a.eb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+parallel = 1
+separate_build_dir = True
+configopts = '-DCMAKE_BUILD_TYPE=Release '
```
Comment (akesandgren) [inline easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-intel-2020a.eb]: Use build_type = 'Release' instead

File: easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-foss-2020a.eb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+parallel = 1
+separate_build_dir = True
+configopts = '-DCMAKE_BUILD_TYPE=Release '
```
Comment (akesandgren) [inline easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-foss-2020a.eb]: Use build_type = 'Release' instead

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/9feb2df54703872d4a4d331d6896e141 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/d60a7e0b5719e5bd152c10b815819e47 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/02efd2dcdd4ae9365b5729e439881e8d for a full test report.

File: easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-foss-2020a.eb [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+parallel = 1
+separate_build_dir = True
+build_type = 'Release'
+
+modextravars = {
+    'KIM_API_CMAKE_PREFIX_DIR': '%(installdir)s/lib64'
+}
+
+sanity_check_paths = {
+    'files': ['bin/kim-api-collections-management', 'lib64/libkim-api.so'],
```
Comment (akesandgren) [inline easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-foss-2020a.eb]: Missed this one, use SHLIB_EXT

File: easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-intel-2020a.eb [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+parallel = 1
+separate_build_dir = True
+build_type = 'Release'
+
+modextravars = {
+    'KIM_API_CMAKE_PREFIX_DIR': '%(installdir)s/lib64'
+}
+
+sanity_check_paths = {
+    'files': ['bin/kim-api-collections-management', 'lib64/libkim-api.so'],
```
Comment (akesandgren) [inline easybuild/easyconfigs/k/kim-api/kim-api-2.1.3-intel-2020a.eb]: Same here, SHLIB_EXT

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/b14e1fe3e46a62ce31b6fb0c286989ad for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @ocaisa!

**Pull request merged**